docs/sources/configure-client/grafana-agent/java/_index.md

```diff
@@ -75,37 +71,32 @@ see [profiler-options](https://github.com/async-profiler/async-profiler?tab=read
 
 ### Set privileges for the collector
 
-You must run the collector, either Grafana Alloy (recommended) or Agent (legacy), as root and inside host `pid` namespace for the `pyroscope.java`
-and `discover.process` components to work.
+You must run the collector, either Grafana Alloy as root and inside host `pid` namespace for the `pyroscope.java` and `discover.process` components to work.
```

Comment:
> either Grafana Alloy as root 

something is off here, we don't need  "either"

Reply:
Also running Alloy as root is the default expectation Running as non-root is a special case (there are docs for setting up non-root)

Reply:
Good catch.